scaling_performance/optimizing_compute_resources.adoc

```diff
@@ -0,0 +1,106 @@
+Administration Guide].
+
+[[scaling-performance-image-considerations]]
+== Image Considerations
+
+[[scaling-performance-predeployed-image]]
+=== Using a Pre-deployed Image to Improve Efficiency
+
+You can create a base {product-title} image with a number of tasks built-in to
+improve efficiency, maintain configuration consistency on all node hosts, and
+reduce repetitive tasks. This is known as a pre-deployed image.
+
+For example, because every node requires the `ose-pod` image in order to run
+pods, each node has to periodically connect to the Docker registry in order to
+pull the latest image. This can become problematic when you have 100 nodes
+attempting this at the same time, and can lead to resource contention on the
+image registry, waste of network bandwidth, and increased pod launch times.
+
+To build a pre-deployed image:
+
+* Create an instance of the type and size required.
+* Ensure a dedicated storage device is available for Docker local image or container storage, separate from any persistent volumes for containers.
+* Fully update the system, and ensure Docker is installed.
+* Ensure the host has access to all yum repositories.
+* link:https://access.redhat.com/documentation/en/red-hat-enterprise-linux-atomic-host/7/getting-started-with-containers/chapter-8-managing-storage-with-docker-formatted-containers[Set up thin-provisioned LVM storage].
+* Pre-seed your commonly used images (such as the *rhel7* base image), as well as
+{product-title} infrastructure container images (*ose-pod*, *ose-deployer*,
+etc.) into your pre-deployed image.
+
+Ensure that pre-deployed images are configured for any appropriate cluster
+configurations, such as being able to run on
+xref:../install_config/configuring_openstack.adoc#install-config-configuring-openstack[OpenStack],
+or
+xref:../install_config/configuring_aws.adoc#install-config-configuring-aws[AWS],
+as well as any other cluster configurations.
+
+[[scaling-performance-prepulling-images]]
+=== Pre-pulling Images
+
+To efficiently produce images, you can pre-pull any necessary container images
+to all node hosts. This means the image does not have to be initially pulled,
+which saves time and performance over slow connections, especially for images,
+such as xref:../creating_images/s2i.adoc#creating-images-s2i[S2I], metrics, and logging, which can be large.
+
+This is also useful for machines that cannot access the registry for security
+purposes.
+
+Alternatively, you can use a local image instead of the default of a specified registry. To do this:
+
+. Pull from local images by setting the `imagePullPolicy` parameter of a pod configuration to `IfNotPresent` or `Never`.
+
+. Ensure that all nodes in the cluster have the same images saved locally.
+
+[NOTE]
+====
+Pulling from a local registry is suitable if you can control node configuration,
+and will not work reliably on cloud providers that do not replace nodes
```

Comment:
Seems odd to combine a positive phrase ("is suitable") with a negative one (will not work). Split into two sentences. (?)